src/PanelsView.tsx

```diff
@@ -21,28 +23,50 @@ import { useSourceActor } from './sourceMachine';
 import { SpinnerWithText } from './SpinnerWithText';
 import { StatePanel } from './StatePanel';
 
-export const PanelsView = () => {
+export const PanelsView = (props: BoxProps) => {
+  const embed = useEmbed();
   const simService = useSimulation();
   const authService = useAuth();
   const [sourceState, sendToSourceService] = useSourceActor(authService);
 
   return (
-    <ResizableBox gridArea="panels" minHeight={0}>
+    <ResizableBox
+      {...props}
+      gridArea="panels"
+      minHeight={0}
+      hidden={embed.isEmbedded && embed.mode === 'viz'}
+    >
       <Tabs
         bg="gray.800"
         display="grid"
         gridTemplateRows="3rem 1fr"
         height="100%"
+        defaultIndex={embed.panelIndex}
```

Comment:
`panelIndex` is only used here - IMHO it would make sense to keep it local and just compute/derive it here